pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/dashboard/DisplayType.java

```diff
@@ -0,0 +1,10 @@
+package org.broadinstitute.dsm.model.dashboard;
+
+public enum DisplayType {
+    HORIZONTAL_BAR_CHART,
+    VERTICAL_BAR_CHART,
+    VERTICAL_HIGHLIGHTED_BAR_CHART,
+    DONUT_CHART,
+    PIE_CHART
```

Comment:
that one is not in `ChartStrategyFactory` is that ok?

Reply:
yup, just kept it for an example, if there will be a `PIE` chart, it will be easy to guess how to follow the structure easily.